apps/arduino_module/module_test/module_test.ino

```diff
@@ -0,0 +1,50 @@
+#include <Arduino.h>
+
+#include <Wire.h>
+#include <stdint.h>
+#include "signpost_api.h"
```

Comment:
A little nit-picky, but it's nice to have headers clustered by where they come from, usually it's system headers, then platform, then local. As the "demo" file it's nice to have this right for people to copy from. So this would like like:

```c
#include <stdint.h>

#include <Arduino.h>
#include <Wire.h> 

#include "signpost_api.h"
```